lib/ansible/plugins/connection/ssh.py

```diff
@@ -279,9 +279,9 @@
           - name: ansible_control_path
             version_added: '2.7'
       control_path_dir:
-        default: ~/.ansible/cp
         description:
           - This sets the directory to use for ssh control path if the control path setting is null.
+          - If null (default), it defaults to ``$ANSIBLE_HOME/cp``.
```

Comment:
Ok. I’ll fix both shortly, including in the changelog. 

Is there a convention for referring to the config setting in documentation? Is the current `$ANSIBLE_HOME/cp` acceptable here, given that it does not call it an environment variable? `{{ANSIBLE_HOME}}/cp`? `the “cp” subdirectory within ANSIBLE_HOME`?

Reply:
Yes, also I'm hoping to make that 'just work' in the future while assigned to `default:` for plugins as it currently works for the base config.

Reply:
Makes sense. You can see in the [history](https://github.com/ansible/ansible/pull/82026/commits/1f32b8d82b533aefb8c0daa764f011d3a32567c8#diff-38cec806ea1a1ee7c3a286c7865334ecfba7ccc49e21d4e8fb8ec1b17938fda6R282), I tried that initially and it worked on like 80% of the tests... I'm guessing it was trying to pull from the env var instead of the config setting?

Reply:
No, it has no way to pull the env in that context, var nor understand `$VAR` syntax

Reply:
Ah, ok.

With the latest revisions, is this PR good to go?